template/CordovaApp.Phone.jsproj

```diff
@@ -79,7 +79,9 @@
     <AppxManifest Include="package.phone.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
-    <Content Include="images\*.png" Exclude="images\*.scale-180.*" />
+    <Content Exclude="images\*SplashScreen.*" Include="images\*.png" />
```

Comment:
Oh, I see. I didn't notice first time that you're excluding splashscreens for desktop app